### PR TITLE
fix token don't send on feedback

### PR DESCRIPTION
**Pull request by rsmnarts**
missing token on log push

### CODE REVIEW

**Comment by slimus** [comment]
Can you explain your scenario, please? We have token on fcm notification, for example

**Comment by rsmnarts** [comment]
Before :
Screen Shot 2021-11-04 at 09 21 14

After :
Screen Shot 2021-11-04 at 09 22 29

File: logx/log.go [hunk 1]
```diff
@@ -171,14 +170,14 @@ func GetLogPushEntry(input *InputLog) LogPushEntry {
 	}
 
 	if input.HideToken {
-		token = hideToken(input.Token, 10)
+		input.Token = hideToken(input.Token, 10)
```
Comment (slimus) [inline logx/log.go:173]: I think you don't need to update `input.Token` here, we don't know where we will use this data. 
My suggestion: 

```suggestion
token := input.Token
if input.HideToken {
		token = hideToken(input.Token, 10)
}
```
And use `token` below

What do you think?

Comment (rsmnarts) [reply logx/log.go:173]: Ok, cool. no problem from me

File: logx/log.go [hunk 2]
```diff
@@ -171,14 +170,17 @@ func GetLogPushEntry(input *InputLog) LogPushEntry {
 	}
 
 	if input.HideToken {
```
Comment (slimus) [inline logx/log.go:172]: Extra wrong `if` condition. Let's remove it?

File: logx/log.go [hunk 3]
```diff
@@ -171,14 +170,17 @@ func GetLogPushEntry(input *InputLog) LogPushEntry {
...
+if input.HideToken {
 		token = hideToken(input.Token, 10)
+}
 	}
 
 	return LogPushEntry{
 		ID:       input.ID,
 		Type:     input.Status,
 		Platform: plat,
-		Token:    token,
```
Comment (slimus) [inline logx/log.go:181]: revert `token` here?

**Review by slimus — CHANGES_REQUESTED** [review]

**Comment by slimus** [comment]
And what about tests? Is it possible to write a unit tests here?

**Comment by slimus** [comment]
Much better! LGTM! Thanks @rsmnarts

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
@rsmnarts Thanks.

**Pull request merged**